src/util-device.c

```diff
@@ -451,6 +452,8 @@ void LiveDeviceFinalize(void)
         }
         SCFree(ld);
     }
+
+    TAILQ_INIT(&pre_live_devices); // reset the list
```

Comment:
Should it not be done in a separate "reset" function ?

Looks strange to see `SomeFuncFinalize` end with `INIT`